airbnb_project/listings/harvester_app/harvester/custom_settings.py

```diff
@@ -1,12 +1,17 @@
+import os
+
 """
 Files contains custom settings for the harvester and values related to AirBnB.
 """
 
 
 def get_scrapy_settings():
+    current_directory = os.path.dirname(os.path.abspath(__file__))
+    # Construct the absolute path to coordinates.json
+    coordinates_file_path = os.path.join(current_directory, 'spiders/listings.csv')
     settings_dict = {
         # Values related to AirBnb
-        'AIRBNB_PUBLIC_API_KEY': "d306zoyjsyarp7ifhu67rjxn52tv0t20",
+        'AIRBNB_PUBLIC_API_KEY': "",
```

Comment:
currently because the harvester isn't really connected to django, this settings file could be seen as an env. Once I had this to django, I will use an env file.

Reply:
ok, this can go in next PR then.